doc/task_breakdown.md

```diff
@@ -0,0 +1,54 @@
+# SQL Query Builder Task Breakdown
+
+## 1. Table Management
+- Add a list of tables from the database.
+- Add functionality for users to select one or more tables for the query.
+- Add functionality for users to remove tables from the selection.
+- Implement removal of related joins, filters, sorting and grouping options.
+- Implement dynamic updating of the lists of available and columns.
+---
+
+## 2. Connection Management
```

Comment:
Thats strange that every time you use different word that mean "table join" 
If you really want to make joins as separate part of your tasks, you can call it Join management

In normal situation "connection" in DB app mean connection to the DB